codespell_lib/data/dictionary.txt

```diff
@@ -9030,6 +9029,7 @@ fysisist->physicist
 fysisit->physicist
 gadged->gadget, gauged,
 gae->game, Gael, gale,
+gage->guage
```

Comment:
Suggest gag

Reply:
Also "gauge"

Reply:
> Also "gauge"

Doh, I think guage is a misspelling too, but I didn't clock it.

Reply:
Indeed Travis caught it even if I didn't `AssertionError: error gage: correction guage is an error itself`